### PR TITLE
HPCC-15090 Simplify Timing Map Labels

### PR DESCRIPTION
**Pull request by GordonSmith**
Fixes HPCC-15090

Signed-off-by: Gordon Smith gordonjsmith@gmail.com

### CODE REVIEW

**Comment by GordonSmith** [comment]
@miguelvazq Please Review

File: esp/src/eclwatch/TimingTreeMapWidget.js [hunk 1]
```diff
@@ -211,7 +211,9 @@ define([
                     this.stdDev = Math.sqrt(variance);
                     for (var i = 0; i < timers.length; ++i) {
                         var prefix = "other";
-                        if (timers[i].Name.indexOf("Graph graph") == 0) {
+                        timers[i].__hpcc_name = timers[i].Name;
+                        if (timers[i].Name.indexOf("Graph graph") === 0) {
```
Comment (miguelvazq) [inline esp/src/eclwatch/TimingTreeMapWidget.js]: Within String.prototype.IndexOf I do not think you can check for multiple instances of a string. The first parameter is the search value and the second _optional_ is the index. Also, zero in this case is if it _does exist_ -1 if it _does not exist_

Comment (GordonSmith) [inline esp/src/eclwatch/TimingTreeMapWidget.js]: I am only checking if the string start with "Graph graph".  I am not checking if the string contains "Graph" or "graph"

Comment (miguelvazq) [inline esp/src/eclwatch/TimingTreeMapWidget.js]: OK graph being a subgraph I'm assuming. If that is the case it looks fine.

**Pull request merged**